schema/iiif_3_0.json

```diff
@@ -579,16 +685,22 @@
                 "behavior": { "$ref": "#/classes/behavior" },
                 "items": {
                     "type": "array",
-                    "items": { "$ref": "#/classes/canvas" }
+                    "items": {
+                      "oneOf": [
+                        { "$ref": "#/classes/manifest" },
+                        { "$ref": "#/classes/canvas" },
+                        { "$ref": "#/classes/collection" }
+                      ]
```

Comment:
I like what you've done here but I think its re-using too much. I think the allOf on line 643 should be a oneOf either manifest or collection (and in the future annotaitonCollection) so something like:

"oneOf": [
  { "$ref": "#/classes/manifest" },
  { "$ref": "#/classes/collection" }
]

Reply:
I have made most of the changes.  But wanted to touch base on this issue here and have you look at it before I look at the permissions below.   I removed most of the duplication, I think.  It required me to add a line to one of your fixtures to add an id to the body of an annotation.  I am not sure if this is actually a required field or not but I think it is.  As a side note if it is required the example I think you copied off of the documentation should be updated to have one.   

Reply:
Looking into this it looks like its only a _may_ for body ids when its a textual body:

"The IRI that identifies the Textual Body.
The Body MAY have exactly 1 IRI that identifies it."

From https://www.w3.org/TR/annotation-model/#classes "3.2.4 Embedded Textual Body" so I think the example is OK and we should fix the validator to make the annotation body optional for Text bodies.